src/locale/locales/it/messages.po

```diff
@@ -75,24 +64,14 @@ msgstr ""
 msgid "{0, plural, one {Reply (# reply)} other {Reply (# replies)}}"
 msgstr ""
 
-#: src/view/com/post-thread/PostThreadItem.tsx:338
+#: src/view/com/post-thread/PostThreadItem.tsx:339
 msgid "{0, plural, one {repost} other {reposts}}"
 msgstr ""
```

Comment:
```suggestion
msgstr "{0, plural, one {ripubblicazione} other {ripubblicazioni}}"
```
And the same here?

Reply:
I hadn't translated them because the context of this string wasn't clear to me

Reply:
Yeah it's quite confusing, I could only figure out what's going on by looking at what translators have done for other languages 😅